scipy/special/sf_error.cc

```diff
@@ -1,8 +1,3 @@
-#include <Python.h>
-
-#include <stdarg.h>
-#include <stdlib.h>
```

Comment:
This does not look correct. At least `stdarg` is needed for `va_start` & co. I don't immediately see if `stdlib` is used too.
Also @steppi why does this use `<stdarg.h>` rather than `<cstdarg>`?

Reply:
The next four includes are these three and one more; I found this when attempting to check for duplicate includes.

Reply:
Right, I just looked at the removals. Sorry for the noise - resolving.